pkg/controllers/vdb/verticadb_controller.go

```diff
@@ -124,7 +124,7 @@ func (r *VerticaDBReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	if err != nil {
 		return ctrl.Result{}, err
 	}
-	prunner := cmds.MakeClusterPodRunner(log, r.Cfg, passwd)
+	prunner := cmds.MakeClusterPodRunner(log, r.Cfg, vmeta.GetSuperuserName(vdb.Annotations), passwd)
```

Comment:
Can we use `vdb.GetVerticaUser`? It will make it easier to switch if we ever convert the annotation to a CR parm.

EDIT: This is a general comment as I see this pattern in quite a few places.